CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ x.x.x Release notes (yyyy-MM-dd)
 ### Bugfixes
 
 * Use of KVC collection operators on Swift collection types no longer throws an exception.
+* The contents of `List` and `Optional` properties are now correctly preserved when copying
```

Comment:
True, but the entry kind of implies that copying between Realms is the only time this will happen at the moment, which is misleading. You could just add "and other operations resulting in the object graph being recursively traversed from Objective-C" at the end of the sentence. Although most users won't understand the full ramifications of that, if they're affected, they know what to look for.